pyfar/classes/audio.py

```diff
@@ -726,20 +715,14 @@ def fft_norm(self, value):
             raise ValueError(("Invalid FFT normalization. Has to be "
                               f"{', '.join(self._VALID_FFT_NORMS)}, but found "
                               f"'{value}'"))
-
-        # apply new normalization if Signal is in frequency domain
-        if self._fft_norm != value and self._domain == 'freq':
-            # de-normalize
-            self._data = fft.normalization(
-                self._data, self._n_samples, self._sampling_rate,
-                self._fft_norm, inverse=True)
-            # normalize
-            self._data = fft.normalization(
-                self._data, self._n_samples, self._sampling_rate,
-                value, inverse=False)
-
         self._fft_norm = value
 
+    @property
```

Comment:
Suggestion: Lets move this up right below the `freq`property to make the concept more clear is someone sees the code for the first time

Question: Don't we need a setter for `freq_raw`? I thought it would be the easiest way if we change `freq_raw`, e.g., in the DSP module to also put it back in to the Signal object using a corresponding setter.

Reply:
Good point, moved it.
I added a setter for `freq_raw` and removed the normalization in __arithmetic_.
Which positions in the code do you refer to in the dsp module?

Reply:
I was not referring to any specific position in DSP. I thought it was the next step, to check where we should work with `freq_raw` in follow up Pull requests.